stable/telegraf/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.service.account.enabled }}
```

Comment:
See https://helm.sh/docs/topics/chart_best_practices/rbac/

Reply:
@davidkarlsen updated it, please let me know if I missed anything